programs/clearing_house/src/math/amm.rs

```diff
@@ -1435,16 +1464,217 @@ pub fn calculate_max_base_asset_amount_fillable(
     )
 }
 
+pub fn calculate_net_user_pnl(amm: &AMM, oracle_price: i128) -> ClearingHouseResult<i128> {
+    let net_user_base_asset_value = amm
+        .net_base_asset_amount
+        .checked_mul(oracle_price)
+        .ok_or_else(math_error!())?
+        .checked_div(AMM_RESERVE_PRECISION_I128 * cast_to_i128(PRICE_TO_QUOTE_PRECISION_RATIO)?)
+        .ok_or_else(math_error!())?;
+
+    net_user_base_asset_value
+        .checked_add(
+            amm.quote_asset_amount_long
+                .checked_add(amm.quote_asset_amount_short)
+                .ok_or_else(math_error!())?
+                .checked_sub(amm.cumulative_social_loss)
```

Comment:
you're subtracting a negative number, so increasing pnl? is that right?

Reply:
yes, the idea is the negative pnl from social loss isnt anyone else's positive pnl

so need to remove it